cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1137,3 +1138,25 @@ def test_mixed_security_group_overwrite_validator(head_node_security_groups, que
     )
     expected_message = "make sure.*cluster nodes are reachable" if expect_warning else None
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "root_volume_size, expected_message, ami_size",
+    [
+        (65, None, 50),
+        (25, None, 8),
+        (
+            25,
+            "Root volume size 25 GB must be equal or greater than .* 50 GB.",
```

Comment:
Better to put `the volume size of the AMI .* 50GB.` because the portion to wildcard is only the AMI ID.

Reply:
Using regex makes the test less likely to break if we make some minor change to the error message